src/pages/room/[roomId].tsx

```diff
@@ -17,16 +19,69 @@ const Room: NextPage<RoomProps> = ({ room }) => {
       </Head>
 
       <main>
-        <h1 className="text-primary">{room.name}</h1>
-        <button>CREATE POST</button>
-        <div className="d-flex flex-column">
-          <Link href={`${room.id}/1`}>Thread 1</Link>
-          <Link href={`${room.id}/2`}>Thread 2</Link>
-          <Link href={`${room.id}/3`}>Thread 3</Link>
-          <Link href={`${room.id}/4`}>Thread 4</Link>
-          <Link href={`${room.id}/5`}>Thread 5</Link>
+        <div className="bg-dark vh-100">
+          <div className="d-flex justify-content-between p-2">
+            <h1 className="text-primary">{room.name}</h1>
+            <button className="text-secondary h3">Raumoptionen</button>
+            {/*Popup für Raumoptionen hinzufügen */}
+          </div>
+          <div className="p-2">
+            <button className="btn btn-primary">CREATE POST</button>
+            {/*Popup für Roomcreate hinzufügen */}
+          </div>
+          <div className="p-2">
+            <div className="container-fluid m-0 p-0 w-100">
+              <div className="overflow-auto">
+                <div className="col flex-nowrap">
+                  {threads.length > 0 ? (
+                    threads.map((thread) => (
+                      <div className="col mb-2" key={thread.id}>
+                        <div
+                          className="card text-bg-primary h-100 d-flex align-items-start"
+                          style={{ width: "80%", position: "relative" }}
+                        >
+                          <div className="card-body">
+                            <Link
+                              key={thread.id}
+                              href={`${room.id}/${thread.id}`}
+                            >
+                              <h5 className="card-title text-white text-decoration-underline">
+                                {thread.name}
+                                {" - "}
+                                {thread.creator.name ?? "Unbekannt"}
+                              </h5>
+                              <p className="card-text text-secondary text-decoration-none">
+                                {thread.description ??
+                                  "Dieser Thread hat keine genauere Beschreibung."}
+                              </p>
+                            </Link>
+                          </div>
+                          <button className="btn ms-auto position-absolute top-0 end-0 p-2">
+                            Optionen {/*Popup für Optionen hinzufügen */}
+                          </button>
```

Comment:
```suggestion
                          <Button className="btn-secondary ms-auto position-absolute top-0 end-0 p-2">
                            Optionen {/*Popup für Optionen hinzufügen */}
                          </Button>
```
s.o., hier hab ich einen `btn-secondary` drauß gemacht, da default bei `react-bootstrap` `btn-primary` ist